tests/feedback.test.ts

```diff
@@ -140,11 +145,16 @@ describe('feedback submission', () => {
     const feedbackController = ControllerFactory.feedback(conn);
 
     const submittedFeedbackResponse = await feedbackController.submitFeedback({ feedback }, user);
-    const { uuid } = submittedFeedbackResponse.feedback;
+    const { uuid } = {
+      uuid: submittedFeedbackResponse.feedback,
+    };
```

Comment:
This should be inlined as well, but
```suggestion
    const uuid = submittedFeedbackResponse.feedback;
```